io_export_paper_model.py

```diff
@@ -1226,47 +1320,40 @@ def __init__(self, uvedge, default_width, index, other: UVEdge):
         len_a = min(len_a, (edge.length * sin_b) / (sin_a * cos_b + sin_b * cos_a))
         len_b = 0 if sin_b == 0 else min(sticker_width / sin_b, (edge.length - len_a * cos_a) / cos_b)
 
-        sin, cos = edge.y / edge.length, edge.x / edge.length
+        tangent = edge.normalized()
+        cos, sin = tangent
         self.rot = M.Matrix(((cos, -sin), (sin, cos)))
 
 
-        self.width = sticker_width * 0.9
-
-        raiseleft = M.Matrix(((cos_b, -sin_b), (sin_b, cos_b))) @ edge * len_b / edge.length
-        raiseright = M.Matrix(((-cos_a, -sin_a), (sin_a, -cos_a))) @ edge * len_a / edge.length
 
-        
+        self.width = sticker_width * 0.9
 
-        v3 = UVVertex(second_vertex.co + raiseleft) #go up #1
-        v4 = UVVertex(first_vertex.co) #go down
-        v5 = UVVertex((15*first_vertex.co + 85*second_vertex.co)/100 +raiseright) #stay up #1
-        v6 = UVVertex((15*first_vertex.co + 85*second_vertex.co)/100) #go down #1
-        v7 = UVVertex((25*first_vertex.co + 75*second_vertex.co)/100) #stay down #1
+        tab = svg2uv("C:\Program Files\\Blender Foundation\\Blender 2.81\\2.81\\scripts\\addons\\Stickers\\tooth-sawtooth.svg")
 
-        v8 = UVVertex((25*first_vertex.co + 75*second_vertex.co)/100 + raiseleft) #go up #2
-        v9 = UVVertex((40*first_vertex.co + 60*second_vertex.co)/100 + raiseright) #stay up #2
-        v10 = UVVertex((40*first_vertex.co + 60*second_vertex.co)/100 ) #go down #2
-        v11 = UVVertex((50*first_vertex.co + 50*second_vertex.co)/100 ) #stay down #2
+        tab_verts = []
+        tab_verts_co = []
+        for i in range (len(tab)):
+            vi = UVVertex((second_vertex.co + self.rot@tab[i].co)+ self.rot @ M.Vector((0, self.width * 0.2)))
```

Comment:
this probably needs to be changed to a matrix rotation on an OBJECT rather than a vertex-by-vertex rotation, but tbh because this is implemented as a local class (ie. not a blender built-in)

Reply:
note that a list of points can be converted to a list of vectors iff the list is well-ordered, but the SVG construction won't guarantee this -- UNLESS the parser guarantees this, so also tbd

Reply:
self.rot @ M.Vector((0, self.width * 0.2)) -- make sure we can justify this